CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 #############################################################################
 # NAME AND VERSION
 ###############################################################################
+
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
This setting should probably be placed in CommonCMake.txt, along with the other compiler options.